### DISCUSSION TITLE
Support Microsoft Entra ID (previous Azure AD)

### DISCUSSION THREAD

**Discussion by ShaniHarush** [kubernetes-sigs/headlamp#2712] [Q&A]
Hi,

I am trying to use  Microsoft Entra ID to log in to Headlamp. Is it possible?

**Comment by illume** [comment]
There was support for this merged in https://github.com/kubernetes-sigs/headlamp/pull/3177 and it will be in the next release. In there there's also some documentation.

A community member wrote a tutorial on how to do it using a reverse proxy here: https://soapfault.com/2025/03/01/azure-aks-entra-oidc-rbac-headlamp/
